feat(changelog): functions-changed-nodejs-20-is-available 2023-04-27 (#1595)
scaleway/docs-content: changelog/april2023/2023-04-27-functions-changed-nodejs-20-is-available.mdx

---
title: NodeJS 20 is available in beta

> status: changed

Reviewer: ```suggestion
status: added
```

author:

>     fullname: 'Join the #[serverless-functions] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #serverless-functions channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2023-04-27
category: serverless
product: functions
---

The runtime list has been updated with:

> - Node20 in beta, it includes the latest version of nodeJs as well as all the improvements we added to node19

Reviewer: ```suggestion
- Node20 in beta. It includes the latest version of nodeJs as well as all the improvements that were added to node19.
```

> - Node19 is now deprecated as it is no longer maintained by the nodeJs team

Reviewer: ```suggestion
```
